terraform/README.md

```diff
@@ -0,0 +1,57 @@
+# SD-Core GNB Integrator Terraform Module
+
+This folder contains a base [Terraform][Terraform] module for the sdcore-gnb-integrator charm.
+
+The module uses the [Terraform Juju provider][Terraform Juju provider] to model the charm
+deployment onto any Kubernetes environment managed by [Juju][Juju].
```

Comment:
`onto any Kubernetes environment`
I think that according to the convention, the repo name should be `sdcore-gnb-integrator-k8s-operator` then. Charm name in `metadata.yaml` should also get the `-k8s` suffix I believe.

Reply:
Agree

Reply:
I was basing the name off other integrator charms, such as [S3 integrator](https://github.com/canonical/s3-integrator/blob/main/metadata.yaml)